runelite-client/src/main/java/net/runelite/client/plugins/chatnotifications/ChatNotificationsConfig.java

```diff
@@ -101,4 +101,26 @@ default boolean notifyOnDuel()
 	{
 		return false;
 	}
+
+	@ConfigItem(
+		position = 6,
+		keyName = "ignoreSelfNotifications",
+		name = "Ignore own messages",
+		description = "Ignores your messages for notifications"
+	)
+	default boolean ignoreSelfNotifications()
+	{
+		return true;
+	}
+
+	@ConfigItem(
+		position = 7,
+		keyName = "ignoreFilteredNotifications",
+		name = "Ignore filtered messages",
+		description = "Ignore filtered messages for notifications"
+	)
+	default boolean ignoreFilteredNotifications()
```

Comment:
Isnt the in-game chat option enough? I would assume you dont want chat notification for anything you dont see in chat.

Reply:
Currently, if you get a filtered message that would be highlighted, you'd get a notification out of nowhere.

Reply:
Yeah exactly. That should not be possibility too imo

Reply:
I'll swap it out for a check for if the player has chat filtered then